tests/cql_query_test.cc

```diff
@@ -3842,3 +3842,81 @@ SEASTAR_TEST_CASE(test_aggregate_and_simple_selection_together) {
         return make_ready_future<>();
     });
 }
+
+SEASTAR_TEST_CASE(test_like_operator) {
+    return do_with_cql_env_thread([] (cql_test_env& e) {
+        cquery_nofail(e, "create table t (p int primary key, s text)");
+        cquery_nofail(e, "insert into t (p, s) values (1, 'abc')");
+        require_rows(e, "select s from t where s like 'abc' allow filtering", {{T("abc")}});
+        require_rows(e, "select s from t where s like 'ab_' allow filtering", {{T("abc")}});
+        cquery_nofail(e, "insert into t (p, s) values (2, 'abb')");
+        require_rows(e, "select s from t where s like 'ab_' allow filtering", {{T("abc")}, {T("abb")}});
+        require_rows(e, "select s from t where s like '%c' allow filtering", {{T("abc")}});
+    });
+}
+
+SEASTAR_TEST_CASE(test_like_operator_on_partition_key) {
+    return do_with_cql_env_thread([] (cql_test_env& e) {
+        // Fully constrained:
+        cquery_nofail(e, "create table t (s text primary key)");
```

Comment:
Please add tests on clustering keys and static columns too, tests with bind markers instead of literals, tests that return no rows, and tests with expected failures (which?)

Reply:
Tests already exist for clustering keys and expected failures.  I'll add the others.